application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Code/MoccaCalendarEventModificationClass.xml

```diff
@@ -208,34 +208,56 @@
       <cache>long</cache>
     </property>
     <property>
-      <code>(function() {
+      <code>require(['moment'], function(moment) {
 
   function disable(element) { element.setAttribute('disabled','disabled') }
   function enable(element) { element.removeAttribute('disabled') }
 
+  function setOriginalDateFormat() {
+    var endDateInput = document.querySelector('input[id^="MoccaCalendar\\.Code\\.MoccaCalendarEventModificationClass_"][id$="_endDate"]');
+    var startDateInput = document.querySelector('input[id^="MoccaCalendar\\.Code\\.MoccaCalendarEventModificationClass_"][id$="_startDate"]');
+    const originalFormat = $('used-format').value;
```

Comment:
does this work? Shouldnt it be:
```suggestion
    const originalFormat = $('#used-format').value;
```

Reply:
I'm not sure why, but if I'm using `$('#used-format')` it does not find the element. I've seen that it was used without the '$' throughout the code. Modified it to `document.getElementById` .